wiki/en/Tips-Tricks-More.md

```diff
@@ -82,11 +82,11 @@ Here is the script:
 
 ## Using ctrlmidich for MIDI controllers
 
-The volume fader, pan control and mute and solo buttons in the client's mixer window strips can be controlled using a MIDI controller by using the `--ctrlmidich` parameter (note: only available for use with macOS and Linux using Jamulus version 3.7.0 or higher, and on Windows using the Jamulus version with JACK support). To enable this feature, Jamulus must be launched with `--ctrlmidich`. There is one global MIDI channel parameter (1-16) and two parameters you can set for each item controlled: `offset` and `consecutive CC numbers`. Set the first parameter to the channel you want Jamulus to listen on (0 for all channels) and then specify the items you want to control (f = volume fader; p = pan; m = mute; s = solo) with the offset (CC number to start from) and number of consecutive CC numbers. Take the following example:
+The volume fader, pan control and mute and solo buttons in the client's mixer window strips can be controlled using a MIDI controller by using the `--ctrlmidich` parameter (note: only available for use with macOS and Linux using Jamulus version 3.7.0 or higher, and on Windows using the Jamulus version with JACK support). To enable this feature, Jamulus must be launched with `--ctrlmidich`. There is one global MIDI channel parameter (1-16) and two parameters you can set for each item controlled: `offset` and `consecutive CC numbers`. Set the first parameter to the channel you want Jamulus to listen on (0 for all channels) and then specify the items you want to control (f = volume fader; p = pan; m = mute; s = solo; o = mute myself) with the offset (CC number to start from) and number of consecutive CC numbers. There is one exception that does not require establishing an offset which is the "Mute Myself" command - it only requires a single CC number as it is only applied to one's own audio stream. Take the following example:
 
-`--ctrlmidich "1;f0*8;p16*8;s32*8;m48*8"`
+`--ctrlmidich "1;f0*8;p16*8;s32*8;m48*8;o64"`
 
-Here, Jamulus listens on MIDI channel 1. Volume fader CC numbers start at 0 and there are 8 of them (so end at CC number 7). Pan controls start at CC number 16 and end at 23; Solo 32 to 39 and Mute 48 to 55.
+Here, Jamulus listens on MIDI channel 1. Volume fader CC numbers start at 0 and there are 8 of them (so end at CC number 7). Pan controls start at CC number 16 and end at 23; Solo 32 to 39 and Mute 48 to 55. Mute Myself is enabled/disabled by CC number 64. Please note that for the functions controlled by buttons to work properly, your MIDI controller needs the buttons to be set to "toggle" mode.
```

Comment:
It might be worth separating
> Please note that for the functions controlled by buttons to work properly, your MIDI controller needs the buttons to be set to "toggle" mode.

into its own paragraph explaining that the controller button should send one value when pressed to "turn on" on the control -- (`>= 64`) -- and a separate value to "turn off" the control -- (`< 64`) -- i.e. "toggle mode".

Reply:
Yes, probably. I'll do that.